daemon/vpn/openvpn/openvpn.go

```diff
@@ -210,6 +211,32 @@ func (ovpn *OpenVPN) stop() error {
 	return nil
 }
 
+func (ovpn *OpenVPN) NetworkChanged() error {
+	// application uses the flag --persist-tun and it should not close the TUN interface at restarts
+	// but because the servers sends different configuration at pull, then the TUN is closed + recreated
+	// https://github.com/OpenVPN/openvpn/blob/a1cb1b47b138b9f654cd0bca5de6d08dbca61888/src/openvpn/init.c#L2421
+
+	return errNotSupported
```

Comment:
IMO, we should say into log that OpenVPN should handle network change by itself, rather that saying it is not supported?